inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_kernel_b_fs_yx_fsv16.cpp

```diff
@@ -190,8 +199,17 @@ JitConstants ConvolutionKernel_b_fs_yx_fsv16::GetJitConstants(const convolution_
 
     ConvolutionTuningData tuning_data = GetTuningParams(params);
 
+    if (input.GetLayout() == DataLayout::b_fs_yx_fsv16 &&
+        output.GetLayout() == DataLayout::bfyx) {
+        jit.AddConstant(MakeJitConstant("OUTPUT_FORMAT_BFYX", 1));
+    }
+
     auto blockWidth = dispatchData.cldnnStyle.blockWidth;
     if (!params.fused_ops.empty()) {
+        DataLayout orig_output_layout = output.GetLayout();
+        if (params.fused_ops.back().GetType() == KernelType::REORDER) {
```

Comment:
if we have post_reorder_fused(), we can use it here

Reply:
Ok, I will.